### PR TITLE
MAGECLOUD-3542: Stabilize functional tests on the Docker

### PR DESCRIPTION
**Pull request by shiftedreality**
### Description

Fixes random fails in Composer cache and Docker volumes.

### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-3542

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: tests/functional/Robo/Tasks.php [hunk 1]
```diff
@@ -18,11 +18,12 @@ trait Tasks
     use TaskAccessor;
 
     /**
+     * @param $volumes
```
Comment (BaDos) [inline tests/functional/Robo/Tasks.php]: please add the type of param

File: tests/functional/Robo/Tasks/EnvUp.php [hunk 2]
```diff
@@ -9,23 +9,50 @@
...
 use Robo\Contract\CommandInterface;
-use Robo\Contract\TaskInterface;
 use Robo\Result;
 use Robo\Task\BaseTask;
+use Magento\MagentoCloud\Test\Functional\Codeception\Docker;
 
 /**
  * Up Docker environment
  */
-class EnvUp extends BaseTask implements CommandInterface, TaskInterface
```
Comment (BaDos) [inline tests/functional/Robo/Tasks/EnvUp.php:19]: Why did you remove `TaskInterface`?

Comment (shiftedreality) [reply tests/functional/Robo/Tasks/EnvUp.php:19]: It is in `BaseTask`

**Review by BaDos — APPROVED** [review]

File: tests/functional/Robo/Tasks.php (hunk 1 above)
Comment (shiftedreality) [reply tests/functional/Robo/Tasks.php]: added

**Comment by shiftedreality** [comment]
Minor change, no additional approve needed.
